src/sentry/incidents/endpoints/serializers.py

```diff
@@ -244,6 +247,38 @@ def validate_target_type(self, target_type):
                 % [item.value for item in AlertRuleTriggerAction.TargetType]
             )
 
+    def validate(self, attrs):
+        if ("target_type" in attrs) != ("target_identifier" in attrs):
+            raise serializers.ValidationError(
+                "targetType and targetIdentifier must be passed together"
+            )
+        target_type = attrs.get("target_type")
+        access = self.context["access"]
+        identifier = attrs.get("target_identifier")
+        if target_type == AlertRuleTriggerAction.TargetType.TEAM:
```

Comment:
Is `target_type` allowed to be null?

Reply:
No, but it might not be provided when performing an update. I actually changed this in https://github.com/getsentry/sentry/pull/15560 because it was hard to reason about, so we require all fields to be passed on update after that pr.